src/CodeStyle/Core/Analyzers/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Diagnostics
+{
+    internal static partial class AnalyzerOptionsExtensions
+    {
+        public static AnalyzerConfigOptions GetOptions(this AnalyzerOptions analyzerOptions, SyntaxTree syntaxTree, CancellationToken _)
```

Comment:
💡 Discards are fine for lambdas and local functions, but exposed signatures of types should always use named parameters.

```suggestion
        public static AnalyzerConfigOptions GetOptions(this AnalyzerOptions analyzerOptions, SyntaxTree syntaxTree, CancellationToken cancellationToken)
```